Update test_pr.md (#2)
jhunshejones-org/maintanance-debugging: test_pr.md

> Modifying the test file to create a new PR

Author: ```suggestion
Modifying the test file again to run status checks
```

> Modifying the test file again to run status checks

Author: ```suggestion
Modifying the test file again to run status checks. Now it's fixed!
```

> Modifying the test file again to run status checks. Now it's fixed!

Author: comment

> Modifying the test file again to run status checks. Now it's fixed!

Author: ```suggestion
Modifying the test file again to run status checks. Now it's fixed for real!
```
